src/main/kotlin/blackjack/model/Player.kt

```diff
@@ -1,16 +1,29 @@
 package blackjack.model
 
+import blackjack.controller.Controller.Companion.INITIAL_CARD_COUNT
+
 abstract class Player(val gamblerInfo: GamblerInfo) {
     private val _cards: MutableList<Card> = mutableListOf()
+    val playerBet: PlayerBet = PlayerBet()
+
     var score: Int = 0
         private set
-
     val cards: List<Card>
         get() = _cards
+    val winnings: Double
+        get() = playerBet.winnings
 
     val name: String
         get() = gamblerInfo.name
 
+    fun setBetAmount(amount: Double) {
+        playerBet.betAmount = amount
+    }
```

Comment:
Instead of making a class with `var` field and a `setXXX`,
why don't we inject it via controller as the object gets created?
I'm thinking something like `Player(gamblerInfo, playerBet)`.
This way, we'll be able to manage them immutably.

This would also mean that `Dealer`'s `calculateAndSetWinnings` would also get changed.